kuma-2.8.yaml

```diff
@@ -0,0 +1,50 @@
+package:
+  name: kuma-2.8
+  version: 2.8.2
+  epoch: 0
+  description: A multi-zone service mesh for containers, Kubernetes and VMs.
+  copyright:
+    - license: Apache-2.0
+  dependencies:
+    provides:
+      - kuma=${{package.full-version}}
+    runtime:
+      - ca-certificates-bundle
+
+var-transforms:
+  - from: ${{package.version}}
+    match: ^(\d+)\.\d+$
+    replace: "$1"
+    to: major-minor-version
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/kumahq/kuma
+      tag: ${{package.version}}
+      expected-commit: ffb0a135b8320cd474c503b785b43627d77a1338
+
+subpackages:
+  - name: kuma-cp-${{vars.major-minor-version}}
+    pipeline:
```

Comment:
We need a:

```
dependencies:
  provides:
    - kuma-cp=${{package.full-version}}
```

Reply:
should this be full or major minor? i thought our names dont include x.y.z

Reply:
the name should be kumbe-cp-{{vars.major-minor-version}}, but it should specify provides as kuma-cp=${{package.full-version}}

Reply:
ok, thank you!